### ISSUE TITLE
preventDefault() on pointerdown events don't work as expected on Chrome for Android

### ISSUE THREAD

**Issue by voluntadpear** [facebook/react#17166]
**Do you want to request a feature or report a bug?**
I would like to report a bug.

**What is the current behavior?**
Reproduction CodeSandbox: <https://codesandbox.io/s/pointerdown-prevent-default-react-5f8by>

On Chrome for Android (tested on Chrome 77.0.3865.116):
1. Click on "Without preventDefault() on pointerdown": `pointerdown`, `mousedown`, `focus`, `mouseup` and `click` are triggered.
2. Click on "With preventDefault() on pointerdown": All of the previous events are triggered as well.

The [Pointer Events spec](https://www.w3.org/TR/pointerevents/#the-pointerdown-event) specifies that canceling the `pointerdown` event (if the `isPrimary` property is `true`) prevents the firing of certain compatibility mouse events (i.e. `mouseup` and `mousedown`).

On a React app, this work as expected on the current versions of the desktop apps I tested. It also works as expected on iOS Safari.

**What is the expected behavior?**
Clicking on "With preventDefault() on pointerdown": Should only trigger `pointerdown` and `click` events.

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**

Tested on React 16.10.2 on Chrome for Android (Chrome 77.0.3865.116). Didn't work on React 16.8.6 either.

**Comment by philipp-spiess** [comment]
I suspect this is a browser issue. Did it work on Chrome for Android before? (I don't have a device to test unfortunately)

There's no special code in React for handing `preventDefault()` of pointer events. We simply forward the call to the native event: https://github.com/facebook/react/blob/5faf377df5267c4248599e14311a75c2f46050c0/packages/legacy-events/SyntheticEvent.js#L115-L128

Do you mind checking what happens when you use native event listeners instead? https://codesandbox.io/s/pointerdown-prevent-default-react-xnydk

If the same issue occurs, I suggest we file a Chromium ticket instead 🙂

**Comment by voluntadpear** [comment]
I made a pure vanilla CodeSandbox and it happens there as well: https://codesandbox.io/s/pointerdown-prevent-default-vanilla-o8265

I did some tests and discovered that this only happens when the target element is part of an `iframe`. In the case of CodeSandbox, all the user code is part of an iframe so I took my vanilla example and tried it outside CodeSandbox and it works just fine. Then I put the example inside an iframe and the wrong behavior happens again.

I will close this issue since it's not React-related, as you pointed out. 🙂
